NuRadioMC/SignalGen/emitter.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import h5py
+from scipy.interpolate import interp1d
+from NuRadioReco.utilities import units, fft
+import logging
+logger = logging.getLogger("SignalGen.emitter")
+import os
+
+def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
+    """
+    returns the electric field of an emitter
+
+    We implement only the time-domain solution and obtain the frequency spectrum
+    via FFT (with the standard normalization of NuRadioMC). This approach assures
+    that the units are interpreted correctly. In the time domain, the amplitudes
+    are well defined and not details about fourier transform normalizations needs
+    to be known by the user.
+
+    Parameters
+    ----------
+    energy : float
+        energy of the shower
+    theta: float
+        viewangle: angle between shower axis (neutrino direction) and the line
+        of sight between interaction and detector
+    N : int
+        number of samples in the time domain
+    dt: float
+        time bin width, i.e. the inverse of the sampling rate
+    shower_type: string (default "HAD")
+        type of shower, either "HAD" (hadronic), "EM" (electromagnetic) or "TAU" (tau lepton induced)
+        note that TAU showers are currently only implemented in the ARZ2019 model
+    n_index: float
+        index of refraction at interaction vertex
+    R: float
+        distance from vertex to observer
+    model: string
+        specifies the signal model
+        * spherical: a simple signal model of a spherical delta pulse emitter
```

Comment:
This doesn't seem right. Can you remove the parameters that are not used, and complete the list of available models?